setup.py

```diff
@@ -52,10 +52,11 @@
     author="MIT Department of Aeronautics and Astronautics",
     author_email="gpkit@mit.edu",
     url="https://www.github.com/hoburg/gpkit",
-    install_requires=["numpy", "scipy"],
-    version="0.3.4.9",
-    packages=["gpkit", "gpkit._mosek", "gpkit.tests", "gpkit.interactive"],
-    package_data={"gpkit": ["gpkit_*.txt", "env/*"],
+    install_requires=["numpy", "scipy", "pint"],
+    version="0.3.6.0",
```

Comment:
I suppose so -- it would be nice for 0.4.0 to be polished, but I'm thinking the refactor introduces enough changes (including non-backwards-compatibility due to removing `setup`) that it ought to trigger the switch from 0.3 to 0.4.

Willing to be told that's a bad idea though -- I don't know how guidelines for releases work.